pkg/proxy/util/utils.go

```diff
@@ -258,27 +214,29 @@ func MapCIDRsByIPFamily(cidrStrings []string) map[v1.IPFamily][]string {
 	return ipFamilyMap
 }
 
-func getIPFamilyFromIP(ipStr string) (v1.IPFamily, error) {
+// Returns the IP family of ipStr, or "" if ipStr can't be parsed as an IP
+func getIPFamilyFromIP(ipStr string) v1.IPFamily {
```

Comment:
Not a big fan of this change, but I guess it doesn't really matter in practice.

Reply:
I considered having it return `(v1.IPFamily, bool)` but it seemed pointless since the second value is completely redundant with the first, and it's not an exported API...